hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/connector/mongodb/MongoStreamSqlConnectorTest.java

```diff
@@ -97,8 +98,14 @@ public void readsFromMongo(boolean includeIdInMapping, boolean forceTwoSteps) {
                     + "    'startAt' = 'now' "
```

Comment:
Wouldn't it be better not to use `now` here instead of the latch?

Reply:
actually you are right <facepalm>

I've committed this change.